composer.json

```diff
@@ -8,16 +8,16 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "~8.2.0",
+        "php": "^8.2",
```

Comment:
Can you please undo this change? The site runs on 8.2 so that should be enforced.

Reply:
done 👍🏽